index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="src/style.css">
+    <title>About RealCoolKids</title>
+</head>
+<body>
+    <h1>About RealCoolKids</h1>
+    <p>These are the RealCoolKids, and this is a list of their favorite things.</p>
+    <ul>
+        <li><a href="#Jack">Jack</a></li>
+        <li><a href="#Denise">Denise</a></li>
+        <li><a href="#Erin">Erin</a></li>
+        <li><a href="#Leigh-Ann">Leigh-Ann</a></li>
+    </ul>
+    <hr>
+    <div id="Jack">
+        <h2>Jack</h2> 
+        <img src="assets/skiing.jpg" alt="waterskiing">
+        <p>I waterskied in college on a competitive team.</p>
+        <h3>These are my favorite things:</h3>
+        <ol>
+            <li>watersking</li>
+            <li>lacrosse</li>
+            <li>hockey</li>
+        </ol>
+    </div>
+    <hr>
+    <div id="Denise">
+        <h2>Denise</h2> 
+        <img src="assets/mountains.jpeg" alt="mountains">
+        <p>Planning my cabin in the woods.</p>
+        <h3>These are my favorite things:</h3>
+            <ol>
+                <li>tacos</li>
+                <li>running</li>
+                <li>swimming</li>
+            </ol>
+    </div>
+    <hr>
+    <div id="Erin">
+        <h2>Erin</h2> 
+        <img src="assets/bear.jpeg" alt="bear">
+        <p>Bears are my spirit animal!</p>
+        <h3>These are my favorite things:</h3>
+        <ol>
+            <li>poetry</li>
+            <li>traveling</li>
+            <li>nature</li>
+        </ol>
+    </div>
+    <hr>
+    <div id="Leigh-Ann">
+        <h2>Leigh-Ann</h2> 
+        <img src="assets/neon.jpeg" alt="neon">
+        <p>I studied glass bending in college.</p>
+        <h3>These are my favorite things:</h3>
+        <ol>
+            <li>eurovision</li>
+            <li>puppies</li>
+            <li>podcasts</li>
+        </ol>
+    </div>
+  
```

Comment:
A footer tag is always a good idea! <footer>By Leigh-Ann, Erin, Denise, & Jack</footer> <- include links to your GitHub?